City-of-Helsinki/drupal-hdbt
# UHF-1049: Center heros text content with text on left/right, move top…

… hero koro one pixel down to avoid leak

How to test:
- `composer require drupal/hdbt:dev-UHF-1049-hero-fixes`
- `make drush-cr`
- Go to https://helfi.docker.sh/fi/hero-kuva-oikealla or https://hel-platform.docker.sh/fi/hero-kuva-oikealla depending on which site you are using.
- Check that the hero text content is vertically aligned to center.
